tool/tsh/tsh.go

```diff
@@ -2669,9 +2669,13 @@ func makeClientForProxy(cf *CLIConf, proxy string, useProfileLogin bool) (*clien
 		return nil, trace.Wrap(err)
 	}
 
-	// apply defaults
+	// apply defaults (by default only issue a certificate for 1 hour)
 	if cf.MinsToLive == 0 {
-		cf.MinsToLive = int32(apidefaults.CertDuration / time.Minute)
+		cf.MinsToLive = int32(60)
+	}
+
+	if (cf.MinsToLive > int32(apidefaults.CertDuration / time.Minute)) {
+		return nil, trace.BadParameter("Maximum ttl allowed is 12hours (720 minutes)")		
```

Comment:
As per the description it seems that we are only allowing the max is 8 hrs for cert ttl. While the description is still 12 hrs.

Reply:
idemeum sessions can be set by admins from: 8hrs, 1 day, 7 days, 30 days. We don't know what is the max that we value the admins sets. If they choose idemeum session to 1 days then still only allow a maximum of 12h for the ssh cert.